core/admin/mailu/ui/forms.py

```diff
@@ -39,7 +39,7 @@ def __init__(self,message=_('Invalid email address.')):
 
     def __call__(self, form, field):
         pattern = re.compile(r'^([_a-z0-9\-\+]+)(\.[_a-z0-9\-\+]+)*@([a-z0-9\-]{1,}\.)*([a-z]{1,})(,([_a-z0-9\-\+]+)(\.[_a-z0-9\-\+]+)*@([a-z0-9\-]{1,}\.)*([a-z]{2,}))*$')
-        if not pattern.match(field.data.replace(" ", "")):
+        if not pattern.match(field.data.lower().replace(" ", "")):
```

Comment:
I'd rather do the check in the regular expression. Removing all spaces is not exactly correct. 
```

        pattern = re.compile(r'^ *([_a-z0-9\-\+]+)(\.[_a-z0-9\-\+]+)*@([a-z0-9\-]{1,}\.)*([a-z]{1,})( *, *([_a-z0-9\-\+]+)(\.[_a-z0-9\-\+]+)*@([a-z0-9\-]{1,}\.)*([a-z]{2,}))* *$', re.I)
        if not pattern.match(field.data):
```

This will not validate idn findings of course. Only punycode encoded domains will be correct.

Reply:
Well, adding them is not correct either; they may or may not be present... so yes we could have ``\s*`` everywhere. I am sure that none of it matters since ultimately we do store it as a CSV field where we do strip the spaces (so why would we care about them in validation?)

As for the validity of the regexp, it is obviously wrong and not just for IDNs. The localpart can contain all kind of things we currently do not allow. See https://en.wikipedia.org/wiki/Email_address#Local-part ; Here I was trying to fix the reported problem with the simplest patch possible.

Reply:
Sure. I'd say adding re.I would be even simpler.

Reply:
Ok. What about 3057444 then? Or do we want to fix LOCALPART and friends too?